### PR TITLE
theme: remove weblinks translation

### PR DESCRIPTION
**Pull request by jacquerie**
## Description
There's no longer need to translate the description at display time
since a PR against `inspire-dojson` is already doing this translation
once and for all.

## Related Issue
Addresses https://github.com/inspirehep/inspire-dojson/issues/58, depends on...

## Checklist:
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl [hunk 1]
```diff
@@ -173,7 +173,7 @@
             {% do viewInDisplayed.append(1) %}
           {% endif %}
           {{ comma() }}
-          <a href='{{ actual_url }}'>{{ url.get('description') | weblinks }}</a>
```
Comment (jacquerie) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: This is not correct... what's wrong is the translation at display time, but it's ok to have some logic that displays the right thing.

Comment (kaplun) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: I am not understanding if you propose to further amend this code.

Comment (jacquerie) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: I still need to put the right things from DoJSON, so this PR is still WIP!

**Comment by jacquerie** [comment]
This went stale. I will resurrect it when I will implement the missing part in DoJSON (https://github.com/inspirehep/inspire-dojson/issues/58).

**Pull request closed without merging**